api/models/thread.js

```diff
@@ -270,6 +270,30 @@ export const getPublicThreadsByUser = (evalUser: string, options: PaginationOpti
     .run();
 };
 
+export const getLikedThreadsByUser = async (
+  evalUser: string,
+  options: PaginationOptions
+): Promise<Array<DBThread>> => {
+  const { first, after } = options;
+
+  // add filter by type of threadreaction
+  const currentUserThreadReactions = db
+    .table('threadReactions')
+    .getAll(evalUser, { index: 'userId' })
+    .run();
+
+  const threadReactions = await currentUserThreadReactions;
+  const threadIds = threadReactions.map(item => {
+    return item.threadId;
+  });
+
+  return db
+    .table('threads')
+    .getAll(...threadIds)
+    .orderBy(db.desc('lastActive'), db.desc('createdAt'))
```

Comment:
We need to use an index here, I think this is the syntax but I have not used that specific command in a while:

```suggestion
    .orderBy({ index: db.desc('lastActive') })
```